src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+type Props = {
+  tabs: Tab[];
+  onChange: (title: string, content: string) => void;
+  activeTab: string;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, onChange, activeTab }) => {
+  return (
+    <>
+      <ul>
+        {tabs.map((tab) => (
+          <li
+            className={tab.title === activeTab ? "is-active" : ""}
+            data-cy="Tab"
+            key={`#${tab.id}`}
+          >
+            <a
+              data-cy="TabLink"
+              href={`#${tab.id}`}
+              onClick={(evt) => {
+                evt.preventDefault();
+                onChange(tab.title, tab.content);
+              }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here